plugins/modules/site_intent.py

```diff
@@ -90,18 +88,27 @@
               name:
                 description: Name of the floor (eg floor-1).
                 type: str
-              parentName:
-                description: Complete Parent name of the floor to be created(eg Global/USA/San Francisco/BGL_18).
+              parent_name:
+                description: Complete parent name of the floor to be created(eg Global/USA/San Francisco/BGL_18).
```

Comment:
description: Hierarchical parent path of the floor, indicating its location within the site (e.g., "Global/USA/San Francisco/BGL_18").

Reply:
updated.